launcher/verifier/client.go

```diff
@@ -25,6 +25,14 @@ type Challenge struct {
 	ConnID string
 }
 
+// TokenOptions contains fields that will be passed to the Attestation Service TokenOptions field.
+// These fields are used to customize several claims in the token from the Attestation service.
+type TokenOptions struct {
+	CustomAudience string
+	CustomNonce    []string
+	TokenType      string
```

Comment:
nit: can we make the `TokenType` an enum too?

Reply:
We could, but lets handle that as a part of a more fleshed out interface as a whole.
In an ideal world, we would provide a proto or something similar to help people interact with the TEE server.

Lets handle this as a part of that improvement.